plugins/modules/ec2_vol.py

```diff
@@ -104,6 +104,12 @@
       - If set, allows to create volume in an Outpost.
     type: str
     version_added: 3.1.0
+  wait_for_modify_volume_complete:
```

Comment:
I would prefer to add `wait` and `wait_timeout` parameters instead of `wait_for_modify_volume_complete` like most Ansible modules support. For example, https://github.com/ansible-collections/amazon.aws/blob/main/plugins/modules/rds_cluster_snapshot.py#L58.

Reply:
Ok I made a commit to use `wait` and `wait_timeout` instead